.github/workflows/cts_ci.yml

```diff
@@ -149,3 +149,19 @@ jobs:
         with:
           name: build-times-${{ matrix.sycl-impl }}
           path: ${{ env.container-workspace }}/build/build_times.log
+
```

Comment:
`compile-cts` job has following code:
```
    needs: build-image-for-sycl-impl
    # Wait for Docker image builds, but run even if they're skipped
    if: always()
```

I think we might need to check `build-image-for-sycl-impl` job status as well, as `compile-cts` is executed even when `build-image-for-sycl-impl` fails.

Reply:
What you mean is that a CI run should be considered failed when `build-image-for-sycl-impl` fails? In other words, `compile-cts` should run when `build-image-for-sycl-impl` is skipped, but not when it fails?

Reply:
I think so. According to my understanding `build-image-for-sycl-impl` is triggered when we update SYCL implementation version and it might fail (e.g. it builds some implementations from sources and we can fail at this step). I'm not sure if we get valid results from `compile-cts` in such case.

Reply:
Now that I think about it, that can't happen. If the image build fails, the `compile-cts` will still run, but will fail once it tries to pull the new image.

Reply:
I see. Even though there is no job status check in GitHub Actions script, there is a dependency via dockerhub, when both jobs use the same SHA.

Reply:
Exactly. We could probably make this clearer in the workflow definition though, I'll take a note.